### PR TITLE
[frr]: Add staticd process to frr daemon

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: dockers/docker-fpm-frr/start.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ supervisorctl start bgpcfgd
 supervisorctl start zebra
 supervisorctl start staticd
 supervisorctl start bgpd
+supervisorctl start staticd
```
Comment (nikos-github) [inline dockers/docker-fpm-frr/start.sh]: 2 lines above is staticd already.

File: dockers/docker-fpm-frr/supervisord.conf [hunk 2]
```diff
@@ -58,6 +58,17 @@ startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:staticd]
```
Comment (nikos-github) [inline dockers/docker-fpm-frr/supervisord.conf]: Likewise...

Comment (stcheng) [reply dockers/docker-fpm-frr/supervisord.conf]: found it

File: dockers/docker-fpm-frr/start.sh (hunk 1 above)
Comment (stcheng) [reply dockers/docker-fpm-frr/start.sh]: thanks

**Pull request closed without merging**